auth-api/pom.xml

```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>io.stargate</groupId>
     <artifactId>stargate</artifactId>
-    <version>1.0.17-SNAPSHOT</version>
+    <version>1.0.17</version>
```

Comment:
It looks like something is misaligned in the automated version change flow

Reply:
Yep, fixed.